keyboards/stella/stella.c

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2019 HnahKB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "stella.h"
+
+
+void matrix_init_kb(void) {
+  matrix_init_user();
+  led_init_ports();
+};
+
+void led_init_ports(void) {
```

Comment:
This needs to go above `matrix_init_kb`, i think, or you may run into compiler issues

Reply:
I don't think this is problem, I precompile and it completely works

Reply:
`led_init_ports()` is already declared in led.h, and implemented as weak in quantum.c. So yeah, there's no issue here.